src/pointarray.cpp

```diff
@@ -61,6 +61,71 @@ struct OctreeNode
             delete children[i];
     }
 
+/*    void print(const int indent, const bool treeWalkDown, const V3f *const P) const
```

Comment:
Let's uncomment this or delete it.  I'm happy to have debug code compiled as long as it doesn't get called.  At least if it's compiled, then it'll be maintained.